polynote-frontend/package.json

```diff
@@ -6,6 +6,7 @@
     "@types/resize-observer-browser": "0.1.5",
     "@types/tinycon": "0.6.1",
     "@types/uuid": "7.0.0",
+    "@types/css-font-loading-module": "0.0.7",
```

Comment:
Is this necessary? If not, can you please remove it?

Reply:
It needs the type definitions for document.fonts, otherwise it wont compile with typescript.
But i can move it to devDependencies if you want.